src/components/notification/alert-message.ts

```diff
@@ -66,6 +69,25 @@ export function getMessageForAlert({
     return `Target ${url} is back to healthy.`
   }
 
+  const getExpectedMessage = (status: string, responseValue: number) => {
+    if (alert === 'status-not-2xx' && status === 'DOWN') {
```

Comment:
Minor suggestion: having nested ifs (in this case) is clearer/easier to read. For example:
```
if (alert === 'status-not-2xx') {
  if(status === 'DOWN') {
    return `Status is ${responseValue}, was expecting 200.`
  }

  if(status === 'UP') {
    return `Service is ok. Status now 200`
  }
}

if (alert.includes('response-time-greater-than-')){
  const alertTime = parseAlertStringTime(alert)

  if(status === 'DOWN') {
      return `Response time is ${responseValue}ms expecting a ${alertTime}ms`
  }

  if(status === 'UP') {
    return `Service is ok. Response now is within ${alertTime}ms`
  }
}
```